ncm-network/src/main/pan/components/network/types/network/interface.pan

```diff
@@ -34,13 +34,25 @@ type network_interface_alias = {
     Describes the bonding options for configuring channel bonding on EL5 and similar.
```

Comment:
Maybe update comment to remove reference to EL5 (which might make someone think it's out of date) and add ref of https://docs.rs/nmstate/latest/nmstate/struct.BondOptions.html#fields ?